interfaces/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "fuel-client-interfaces"
+version = "0.1.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+repository = "https://github.com/FuelLabs/fuel-core"
+description = "Fuel core interfaces"
+
+[dependencies]
+anyhow = "1.0"
+async-trait = "0.1"
+fuel-asm = "0.1"
+fuel-storage = "0.1"
+fuel-tx = { version = "0.2", features = ["serde-types"] }
+fuel-types = { version = "0.1", features = ["serde-types"] }
+fuel-vm = "0.2"
+futures = "0.3"
+hashbrown = "0.12"
+parking_lot = "0.11"
+thiserror = "1.0"
+tokio = { version = "1.14", features = ["full"] }
```

Comment:
Is tokio needed / can this be moved to a dev dep if so?

Reply:
I am using `tokio::sync::RwLock` inside `TxPoolService`

Reply:
Isn't that a different crate though? I don't think we need tokio to define our interfaces do we?

Reply:
I think it is same crate, but i can minimize features included.